components/grid/export/csv.md

```diff
@@ -5,7 +5,7 @@ description: Export to CSV the Grid for Blazor.
 slug: grid-export-csv
 tags: telerik,blazor,grid,export,csv
 published: True
-position: 1
+position: 3
 ---
 
 # Grid CSV Export
```

Comment:
The Customization section contains mostly information that is not applicable to CSV export.